cmd/extract-block-events/main.go

```diff
@@ -84,7 +84,7 @@ func run() int {
 	}
 	defer db.Close()
 
-	// Initialize the codec we use for the data.
+	// Initialize the codec.
```

Comment:
This is a useless comment after the change. At least put something like `initialize the codec that will be used to encode and decode data in the index database.`